app/src/main/kotlin/com/wire/android/ui/home/conversationslist/AllConversationScreen.kt

```diff
@@ -72,19 +82,20 @@ private fun AllConversationContent(
                 },
                 items = conversationList
             ) { generalConversation ->
-                GeneralConversationItem(
-                    generalConversation = generalConversation,
-                    onConversationItemClick = onConversationItemClick,
-                    onConversationItemLongClick = onEditConversationItem
+                ConversationItemFactory(
```

Comment:
not 100 % sure, but I see a optential problem here with recomposition, as we are not really focusing on making MVP as fast as possible for now :P do not know if it is a big of an issue but : 

similar to above

Reply:
Fixed